client/src/components/ActiveChat/SenderBubble.js

```diff
@@ -24,18 +24,29 @@ const useStyles = makeStyles(() => ({
   bubble: {
     background: "#F4F6FA",
     borderRadius: "10px 10px 0 10px"
-  }
+  },
+  avatar: {
+    height: 20,
+    width: 20,
+    marginRight: 0,
+    marginTop: 6,
+    marginBottom: 6
+  },
 }));
 
 const SenderBubble = (props) => {
   const classes = useStyles();
-  const { time, text } = props;
+  const { time, text, lastReadMessageId, messageId, otherUser } = props;
   return (
     <Box className={classes.root}>
       <Typography className={classes.date}>{time}</Typography>
       <Box className={classes.bubble}>
         <Typography className={classes.text}>{text}</Typography>
       </Box>
+      {lastReadMessageId === messageId && lastReadMessageId !==undefined ? 
+      <Typography >
+        <Avatar alt={otherUser.username} src={otherUser.photoUrl} className={classes.avatar}></Avatar>
+      </Typography>: undefined}
```

Comment:
Why does `Avatar` component need to be wrapped inside `Typography`?

Reply:
You are right, Not needed. Removing